eng/tools/generator/cmd/v2/common/generation.go

```diff
@@ -49,9 +51,11 @@ type GenerateParam struct {
 	RemoveTagSet        bool
 }
 
-func (ctx GenerateContext) GenerateForAutomation(readme, repo, goVersion string) ([]GenerateResult, []error) {
+func (ctx *GenerateContext) GenerateForAutomation(readme, repo, goVersion string) ([]GenerateResult, []error) {
 	absReadme := filepath.Join(ctx.SpecPath, readme)
 	absReadmeGo := filepath.Join(filepath.Dir(absReadme), "readme.go.md")
+	ctx.SpecReadmeFile = absReadme
+	ctx.SpecReadmeGoFile = absReadmeGo
```

Comment:
what if there is no seperate go readme file?

Reply:
It is a must for go mgmt. modules.